### PR TITLE
Add certifications for MP 7.0

### PR DESCRIPTION
**Pull request by benjamin-confino**
Adds certifications for MP70 on 24.0.0.12.

Link to draft site: https://certifications-draft-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/certifications/microprofile/7.0/24.0.0.12-TCKResults.html

### CODE REVIEW

File: microprofile/7.0/24.0.0.12-TCKResults.adoc [hunk 1]
```diff
@@ -0,0 +1,62 @@
+:page-layout: certification
+= MicroProfile 7.0 TCK Results Summary
+
+As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], following is a summary of the MicroProfile TCK results for MicroProfile 7.0.
+
+== Open Liberty 24.0.0.12 MicroProfile 7.0 Certification Summary
+
+MicroProfile 7.0 consists of 8 MicroProfile Specifications plus the Jakarta EE Core Profile 10.0 specfication.
```
Comment (Emily-Jiang) [inline microprofile/7.0/24.0.0.12-TCKResults.adoc]:
```suggestion
MicroProfile 7.0 consists of 7 MicroProfile Specifications and it depends on the Jakarta EE Core Profile 10.0 specfication.
```

**Review by Emily-Jiang — CHANGES_REQUESTED** [review]
The links to the spec and download url should point to the maven central instead of the github repo. e.g. 

https://repo1.maven.org/maven2/org/eclipse/microprofile/fault-tolerance/microprofile-fault-tolerance-spec/4.1/

File: microprofile/7.0/telemetry/2.0/24.0.0.12-MicroProfile-Telemetry-2.0-Java11-TCKResults.adoc [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
++
+https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/runtime/release/24.0.0.12/openliberty-24.0.0.12.zip[Open Liberty 24.0.0.12]
+
+* Specification Name, Version and download URL:
++
+https://repo1.maven.org/maven2/org/eclipse/microprofile/telemetry/microprofile-telemetry-spec/2.0/[MicroProfile Telemetry 2.0 Specification]
+
+* (Optional) TCK Version, digital fingerprint and download URL:
++
+https://repo1.maven.org/maven2/org/eclipse/microprofile/telemetry/microprofile-telemetry-logs-tck/2.0-RC2/microprofile-telemetry-logs-tck-2.0-RC2.jar[MicroProfile Telemetry 2.0 Logs TCK]
```
Comment (Emily-Jiang) [inline microprofile/7.0/telemetry/2.0/24.0.0.12-MicroProfile-Telemetry-2.0-Java11-TCKResults.adoc]: These is the RC not the final jar.

File: microprofile/7.0/telemetry/2.0/24.0.0.12-MicroProfile-Telemetry-2.0-Java11-TCKResults.adoc [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
++
+SHA-1: `2bf36936120751b2fe4b807c020b171e64ebf026`
++
+SHA-256: `8b5672f16d9279f5d6a14097be9428c8102eb9aca67a33dc1d412a956c7b272b`
++
+https://repo1.maven.org/maven2/org/eclipse/microprofile/telemetry/microprofile-telemetry-metrics-tck/2.0-RC2/microprofile-telemetry-metrics-tck-2.0-RC2.jar[MicroProfile Telemetry 2.0 Metrics TCK]
```
Comment (Emily-Jiang) [inline microprofile/7.0/telemetry/2.0/24.0.0.12-MicroProfile-Telemetry-2.0-Java11-TCKResults.adoc]: These is the RC not the final jar.

File: microprofile/7.0/telemetry/2.0/24.0.0.12-MicroProfile-Telemetry-2.0-Java11-TCKResults.adoc [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
++
+SHA-1: `78a23e28f406af8136c0bcc36d38c21cd2bbe066`
++
+SHA-256: `e31ae6d94d0db7fa4e1666bf0ecffbe70987a137a040325c5c32577f508be0ae`
++
+https://repo1.maven.org/maven2/org/eclipse/microprofile/telemetry/microprofile-telemetry-tracing-tck/2.0-RC2/microprofile-telemetry-tracing-tck-2.0-RC2.jar[MicroProfile Telemetry 2.0 Tracing TCK]
```
Comment (Emily-Jiang) [inline microprofile/7.0/telemetry/2.0/24.0.0.12-MicroProfile-Telemetry-2.0-Java11-TCKResults.adoc]: These is the RC not the final jar.

File: microprofile/7.0/telemetry/2.0/24.0.0.12-MicroProfile-Telemetry-2.0-Java11-TCKResults.adoc (hunk 3 above)
Comment (Emily-Jiang) [reply microprofile/7.0/telemetry/2.0/24.0.0.12-MicroProfile-Telemetry-2.0-Java11-TCKResults.adoc]: please make corrections for all Java levels.

File: microprofile/7.0/24.0.0.12-TCKResults.adoc [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/runtime/release/24.0.0.12/openliberty-24.0.0.12.zip[Open Liberty 24.0.0.12]
+
+In order to certify against MicroProfile 7.0, an implementation must pass the TCK tests for the seven included MicroProfile component specifications, plus
+the Jakarta EE Core Profile 10.0 TCK.
+
+
+=== MicroProfile 7.0 Component Specification Certification Summary (Java 11)
+* xref:config/3.1/24.0.0.12-MicroProfile-Config-3.1-Java11-TCKResults.adoc[MicroProfile Config 3.1 TCK Result (Java 11)]
+* xref:faulttolerance/4.1/24.0.0.12-MicroProfile-Fault-Tolerance-4.1-Java11-TCKResults.adoc[MicroProfile Fault Tolerance 4.1 TCK Result (Java 11)]
```
Comment (Emily-Jiang) [inline microprofile/7.0/24.0.0.12-TCKResults.adoc]: wrong link

File: microprofile/7.0/24.0.0.12-TCKResults.adoc [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+* xref:health/4.0.1/24.0.0.12-MicroProfile-Health-4.0.1-Java11-TCKResults.adoc[MicroProfile Health 4.0 TCK Result (Java 11)]
+* xref:jwt/2.1/24.0.0.12-MicroProfile-JWT-Auth-2.1-Java11-TCKResults.adoc[MicroProfile JWT Authentication 2.1 TCK Result (Java 11)]
+* xref:openapi/4.0/24.0.0.12-MicroProfile-Open-API-4.0.2-Java11-TCKResults.adoc[MicroProfile OpenAPI 4.0 TCK Result (Java 11)]
+* xref:restclient/4.0/24.0.0.12-MicroProfile-Rest-Client-4.0-Java11-TCKResults.adoc[MicroProfile Rest Client 4.0 TCK Result (Java 11)]
+* xref:telemetry/2.0/24.0.0.12-MicroProfile-Telemetry-2.0-Java11-TCKResults.adoc[MicroProfile Telemetry 2.0 TCK Result (Java 11)]
```
Comment (Emily-Jiang) [inline microprofile/7.0/24.0.0.12-TCKResults.adoc]: broken link

**Comment by Emily-Jiang** [comment]
@benjamin-confino you made the changes but did not change the links. You need to update the links for all java versions.

**Review by Emily-Jiang — APPROVED** [review]

**Review by ayoho — APPROVED** [review]

**Pull request merged**